src/main/lrsql/admin/interceptors/ui.clj

```diff
@@ -9,20 +9,26 @@
   [_]
   (resp/redirect "/admin/index.html"))
 
-(def get-env
-  "Provide select config data to client upon request"
+(defn get-env
+  "Provide select config data to client upon request. Takes a map with static
+  config to inject:
+    :enable-admin-status - boolean, determines if the admin status endpoint is
+      enabled."
```

Comment:
Weird indentation?

Reply:
I indented 2 spaces so it is clear it is under the list item `:enable-admin-status`. If there are multiple items in the future is clearer IMO

Reply:
Hmm, I actually undid the 2-spaces indentation in other comments because I thought it looked ugly. I had no idea there was an actual reason for that, sorry.

Reply:
No worries, there really isn't an authoritative source on it so 🤷